salt/metalk8s/kubeadm/init/kubeconfig/files/kubeconfig.tpl

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
```

Comment:
Could use `file.serialilze` in your state, instead of a template.

Reply:
OK, I will do

Reply:
@ramanduh is this solved?